inference-engine/src/legacy_api/src/transformations/convert_opset1_to_legacy/fc_bias_fusion.cpp

```diff
@@ -43,7 +43,9 @@ ngraph::pass::FullyConnectedBiasFusion::FullyConnectedBiasFusion() {
         Shape bias_shape(bias->get_shape());
         Shape output_shape(fc->get_shape());
         size_t bias_size = std::accumulate(bias_shape.begin(), bias_shape.end(), size_t{1}, std::multiplies<int64_t>());
-        if (bias_shape.empty() || bias_shape.back() != output_shape.back() || bias_shape.back() != bias_size) {
+        if (bias_shape.empty() ||
+            (bias_shape.back() != output_shape.back() && bias_shape.back() != 1) ||
```

Comment:
I don't fully understand this change. Could you please explain this case?

Reply:
In the case when the bias last dimension is 1 but fully connected layer last output dimension is not 1 broadcast should be applied for the bias but it is still should be fused with fully connected layer. Without this change fusion isn't done for the case where broadcast is required.

Reply:
Added corresponding test